modules/editor/web/js/workspace/explorer.js

```diff
@@ -150,9 +150,9 @@ define(['log', 'jquery', 'backbone', 'lodash', './explorer-item', './service-cli
             activateBtn.attr("data-placement", "bottom").attr("data-container", "body");
 
             if (this.application.isRunningOnMacOS()) {
-                activateBtn.attr("title", "Open file explorer (" + _.get(self._options, 'command.shortcuts.mac.label') + ") ").tooltip();
+                activateBtn.attr("title", "Open File Explorer (" + _.get(self._options, 'command.shortcuts.mac.label') + ") ").tooltip();
```

Comment:
@erangatl shall we format these to 120?

Reply:
This has been completely rewritten in React.